include/seqan3/alignment/pairwise/execution/alignment_executor_two_way.hpp

```diff
@@ -302,11 +300,13 @@ class alignment_executor_two_way
     void move_initialise(alignment_executor_two_way && other) noexcept
     {
         kernel = std::move(other.kernel);
+        _chunk_size = std::move(other._chunk_size);
```

Comment:
Could you remove the leading underscore from the member variable `chunk_size`? I don't see any reason for this and all other members are also without leading `_`.

Reply:
Actually, I'd love to but there is a function called chunk_size as well and that doesn't work. I do not want to invent a new name for the member and rather stick in these rare cases to what most people do extending the name with an underscore.